.vscode/cspell.json

```diff
@@ -89,6 +90,7 @@
     "iusg",
     "jepio",
     "Jeremi",
+    "jnyfah",
```

Comment:
Does it make sense to put these into the changelog directly?  That way we don't have to constantly update the cspell.json file for every contributor.

Something like is done for the readme file in attestation?
```md
<!-- cspell:words opentelemetry -->
```

Reply:
I like the idea of moving cspell exceptions closer to the places where they are applied. For code, that should be a safe way to go.
But in this case, I am not sure if automation that compiles release notes is going to handle the comment correctly.
Let me to complete this PR as it is, and I will open an issue to review and move exceptions to the files.

Reply:
#4668